lib/workers/bin/evm_server.rb

```diff
@@ -1,3 +1,5 @@
+require File.expand_path("../../../config/environment", __dir__)
+
 require "workers/evm_server"
 if MiqEnvironment::Process.is_rails_runner?
   EvmServer.start(*ARGV) 
```

Comment:
LOL, this made me laugh when I first read it and now that I'm revisiting it.  Thanks for the humor.  :laughing: